src/onnx-mlir.cpp

```diff
@@ -68,7 +69,12 @@ int main(int argc, char *argv[]) {
   }
 
   // Create context after MLIRContextCLOptions are registered and parsed.
-  mlir::MLIRContext context;
+  mlir::MLIRContext context(mlir::MLIRContext::Threading::DISABLED);
+  llvm::DefaultThreadPool pool(
```

Comment:
Why do we create a pool with 1 thread when we don't set it? Can we create it only when threads > 1?

Reply:
Thanks for the comment. I thought so too, but when I created the pool within the if statement, I got an error  when using the pool(maybe it was illegal instruction). I'm not sure why this happened.
I will check again.

Reply:
@AlexandreEichenberger  Updated. I think this is better way to create the pool. I studied from LLVM repo.